bower.json

```diff
@@ -4,6 +4,7 @@
   "dependencies": {
     "angular": "1.5.8",
     "angular-animate": "1.5.8",
+    "angular-cookies": "1.5.8",
```

Comment:
BTW there's already https://github.com/tymondesigns/angular-locker which might save you some kb's since no need to add this lib, but I'm fine either way.